src/semck/typeck.rs

```diff
@@ -1827,6 +1827,7 @@ impl<'a, 'ast> MethodLookup<'a, 'ast> {
                 LookupKind::Ctor(cls_id) => {
                     let cls = self.ctxt.classes.idx(cls_id);
                     let cls = cls.read();
+                    println!("adding cstr! {:?}", self.ctxt.interner.str(cls.name));
```

Comment:
please remove this

Reply:
Oh damn, yes, that was a left-over from a different PR. Thanks!